README.md

```diff
@@ -1,2 +1,15 @@
 # crl-janitor
-Janitor service to cleanup resources created by [Cloud Resource Library (CRL)](https://github.com/DataBiosphere/terra-cloud-resource-lib) 
\ No newline at end of file
+Janitor service to cleanup resources created by [Cloud Resource Library (CRL)](https://github.com/DataBiosphere/terra-cloud-resource-lib) 
+
+# Development
+
+## Using the Gradle wrapper
+Set executable permissions:
+```
+chmod +x gradlew
+```
+Start local server
+```
+./gradlew bootRun
```

Comment:
I think we're missing the `gradle/wrapper/gradle-wrapper.jar`. Do you have it locally only?

Reply:
oh right, I ignore .jar files because I saw that is what trd did:
https://github.com/DataBiosphere/jade-data-repo/blob/develop/.gitignore#L17.
Added it back to something like https://github.com/DataBiosphere/terra-workspace-manager/blob/master/.gitignore#L5

Reply:
created https://broadworkbench.atlassian.net/browse/CA-909